src/nomad_material_processing/vapor_deposition/__init__.py

```diff
@@ -62,6 +64,164 @@
 m_package = Package(name='Vapor Deposition')
 
 
+class InsertReduction(ArchiveSection):
+    """
+    The reduction that sometimes is used to lodge the substrate in the substrate holder position..
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    insert_id = Quantity(
+        type=str,
+        description="""
+        The ID of the insert that is placed in this position to accomodate the substrate.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    material = Quantity(
```

Comment:
Shouldn't this material and the one in the SubstrateHolder class be a PubChemPureSubstance section or something like that?